src/core/qgspointlocator.cpp

```diff
@@ -685,7 +694,8 @@ bool QgsPointLocator::rebuildIndex( int maxFeaturesToIndex )
     return true; // nothing to index
 
   QgsFeatureRequest request;
-  request.setSubsetOfAttributes( QgsAttributeList() );
+  //request.setSubsetOfAttributes( QgsAttributeList() );
```

Comment:
Fetching attributes here isn't required, is it? I think this is leftover from an earlier iteration and will cause a performance drop.

Reply:
I add this request in an else